src/block-management/block-holder.js

```diff
@@ -199,12 +200,24 @@ export class BlockHolder extends React.Component<PropsType, StateType> {
 		const borderColor = isSelected ? focusedBorderColor : 'transparent';
 
 		return (
-			<TouchableWithoutFeedback onPress={ this.onFocus } >
+			// Needs to be false to access children
```

Comment:
I'm assuming (and it's also deduced by the link provided) that you're referring to the `accessible` prop here so, let's name/specify it in the comment.

Reply:
Yes that's right, I can mention it in the comment

Reply:
Seems addressed by 7bd3b0b76a5d17a6d023467b07fb687d0f605191.